src/lib/autoinstall/clients/clone_system.rb

```diff
@@ -152,7 +152,14 @@ def clone_system(options)
             Yast::ProductControl.clone_modules + ["general"]
           end
         Yast::AutoinstClone.Process(target: target)
-        Yast::XML.YCPToXMLFile(:profile, Yast::Profile.current, filename)
+        begin
+          Yast::XML.YCPToXMLFile(:profile, Yast::Profile.current, filename)
+        rescue Yast::XMLSerializationError => e
+          log.error "Creation of XML failed with #{e.inspect}"
+          Yast::Popup.Error(
+            _("Invalid object found when creating XML. Please report a bug.")
```

Comment:
np: What about a more user friendly message? "Could not create the XML file. Please, report a bug".